RedHatInsights/insights-advisor-frontend
# Import ReportDetails from advisor-components

### Description

`ReportDetails` component is used to render rule evaluation results for a particular system. The information is divided into four categories: "Detected issues," "Steps to resolve," "Knowledgebase article," and "More info." The PR removes the component implementation from RHEL Advisor and imports the shared one from https://github.com/RedHatInsights/frontend-components/tree/master/packages/advisor-components instead.

### How to test

The best way to ensure no visual change or data loss is to run the PR locally against stage-beta, open another window with console.stage.redhat.com/beta and compare the same system for the equal rule reports.

### Screenshots

![Screenshot 2022-07-08 at 13-56-43 rhiqe-e2e-f675dc64-fd8c-4a5f-86ef-12fecde45e35 test - Systems - Advisor Red Hat Insights](https://user-images.githubusercontent.com/31385370/177988250-cbdbfe24-582a-46ab-8966-8a8c68e8afd6.png)
